### ISSUE TITLE
Query triple digit numbers for anime series with more than 99 episodes

### ISSUE THREAD

**Issue by Taloth** [Sonarr/Sonarr#1125]
https://forums.sonarr.tv/t/dragon-ball-super-episodes-cannot-be-found/9037/5

We could query by both double and triple digits in those cases. Atm the AnimeSearchCriteria doesn't have that information (it only has the series, not the highest abs.nr in the series).

Also we might want to ask nn+ to add a parameter for abs nr with appropriate regex to get better support.

**Comment by markus101** [comment]
The problem for Dragon Ball Super is there aren't yet 100 episodes so we'd still only search `01`, not `001`, I'm in favour of this change, but I do think its use is limited.

**Comment by Taloth** [comment]
Indeed, especially since Dragon Ball Super doesn't even yet have that many episodes.
Querying for triple digits for all anime is even less favorable.

**Comment by gokussx4** [comment]
Actually if you use an indexer like nzbfinder.ws you will find that the episodes when searched with the same criteria that sonarr uses (e.g. "Dragon Ball Super : 30") nothing comes up. If you search with "Dragon Ball Super : 030" you will find the 3 episode of the third season. Not sure why the tv registries are listing them as seasons but sonarr does the search for the correct number. The problem is that a number of indexers have a padded zero on the front for some reason and thus sonarr won't find the result. Haven't looked at the code but if this is just a regex problem then we could probably adjust the regex pattern to include a padded zero. Thoughts?

**Comment by markus101** [comment]
Its not a parsing problem, 3 digit absolute numbers parse fine the issue is knowing what to search for since there is no consistency in which format will be used.

**Comment by bakerboy448** [comment]
For posterity: this likely won't happen anytime soon

**Comment by kiam001** [comment]
Same problem here

Black.Clover.S01E026.German.DL.DTS.1080p.BluRay.x264-***

cant find it search in newst devlop only s01e26 and cant find s01e026
